incydr/cli/core.py

```diff
@@ -40,6 +50,68 @@ def format_help(self, ctx: click.Context, formatter: click.HelpFormatter) -> Non
         )
 
 
+class ExceptionHandlingGroup(IncydrGroup):
```

Comment:
The heart of the exception handling change is in this class.  Open to any/all suggestions for how/what to log for each error if anything jumps out at you!

Reply:
Thoughts on logging the command + args before _any_ error we log, so the logs always show what exactly caused the error?

Reply:
That seems like a good plan